.github/workflows/django-multitenant-tests.yml

```diff
@@ -0,0 +1,80 @@
+on:
+  push:
+    branches:
+      - "**"
+  workflow_dispatch:
+jobs:
+  static-checks:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+    - uses: actions/checkout@v3
+    - name: Install python dependencies
+      run: |
+        pip install -r requirements/static-analysis-requirements.txt
+    - name: Format Checks 
+      run: |
+        black --check --diff .
```

Comment:
What about `make format-checks`? I notice the Makefile commands differ, but you could update the Makefile whatever is best.

Reply:
Fixed as you suggested